modules/rosa-hcp/providers.tf

```diff
@@ -1,23 +1 @@
-terraform {
-  required_version = ">= 1.0"
-
-  required_providers {
-    aws = {
-      source  = "hashicorp/aws"
-      version = ">= 5.35.0"
-    }
-    rhcs = {
-      version = "1.6.6"
-      source  = "terraform-redhat/rhcs"
-    }
-  }
```

Comment:
just for me to understand on why we removed this.
We removed this because we're using the rosa-hcp and vpc module which already defines all of those dependencies and don't add anything else on top?